dso-l2/src/main/java/com/tc/objectserver/impl/DistributedObjectServer.java

```diff
@@ -327,13 +327,17 @@ protected ServerBuilder getServerBuilder() {
     if (pp == null) {
       pp = new MapListPrettyPrint();
     }
-    this.seda.getStageManager().prettyPrint(pp);
-    this.persistor.prettyPrint(pp);
-    this.groupCommManager.prettyPrint(pp);
-    this.l2Coordinator.prettyPrint(pp);
-    this.entityManager.prettyPrint(pp);
-    this.serviceRegistry.prettyPrint(pp);
-    addExtendedConfigState(pp);
+    try {
+      this.seda.getStageManager().prettyPrint(pp);
+      this.persistor.prettyPrint(pp);
+      this.groupCommManager.prettyPrint(pp);
+      this.l2Coordinator.prettyPrint(pp);
+      this.entityManager.prettyPrint(pp);
+      this.serviceRegistry.prettyPrint(pp);
+      addExtendedConfigState(pp);
+    } catch (Throwable t) {
+      pp.println("unable to collect complete cluster state due to " + t.getLocalizedMessage());
+    }
     return pp.toString().getBytes(set);
```

Comment:
I thought that was not necessary as we are already catching exceptions from entity code run in ManagedEntityImpl and TerracottaServiceProviderRegistryImpl and platform code where we add cluster state for other components looks fine. Anyways I will push necessary changes.